pallets/currency/src/lib.rs

```diff
@@ -74,6 +74,18 @@ pub mod pallet {
 			+ Default
 			+ Debug;
 
+		/// Native currency e.g. INTR/KINT
+		#[pallet::constant]
+		type GetNativeCurrencyId: Get<CurrencyId<Self>>;
+
+		/// Relay chain currency e.g. DOT/KSM
+		#[pallet::constant]
+		type GetRelayChainCurrencyId: Get<CurrencyId<Self>>;
+
+		/// Wrapped currency e.g. IBTC/KBTC
+		#[pallet::constant]
+		type GetWrappedCurrencyId: Get<CurrencyId<Self>>;
```

Comment:
What would this mean for us? We will have many "wrapped" assets, how does this play along with our defined `CurrencyId`

Reply:
That's a very good question. I briefly checked the code of the other crates I worked on, and it's not used a lot so I think we might even be able to get rid of it. But I would keep it for now.